atlas/schema.hcl

```
@@ -1,11 +1,430 @@
+  }
+  column "partnership" {
+    null = false
+    type = bool
+    default = false
+    comment = "An attribute of the T2125 form - additional info required if the business is a partnership."
+  }
+  column "address" {
+    null = true
+    type = text
+    comment = "The organization's physical address if applicable."
+  }
+  column "create_time" {
+    null = false
+    type = timestamptz
+  }
+  primary_key {
+    columns = [column.organization_id, column.snapshot_id]
+  }
+  foreign_key "organization_id" {
+    columns     = [column.organization_id]
+    ref_columns = [table.organization_v1.column.organization_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+}
+table "organization_v1" {
+  schema = schema.public
+  comment = "An organization refers to the business that is being filed under. All organization data lives in the organization_snapshot table."
+  column "organization_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  primary_key {
+    columns = [column.organization_id]
+  }
+}
+table "project_v1" {
+  schema = schema.public
+  comment = "A project can be any time a user starts a new form. A user can have many projects."
+  column "user_id" {
+    null = false
+    type = bigint
+  }
+  column "project_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  primary_key {
+    columns = [column.user_id, column.project_id]
+  }
+  foreign_key "user_id" {
+    columns     = [column.user_id]
+    ref_columns = [table.user_v1.column.user_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+index "project_id_idx" {
+    unique  = true
+    columns = [column.project_id]
+  }
+}
+table "scan_inference_v1" {
+  schema = schema.public
+  comment = "This table stores the results of inferencing on the image scans."
+  column "scan_id" {
+    null = false
+    type = bigint
+  }
+  column "scan_inference_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  column "inference_result" {
+    null = false
+    type = jsonb
+    comment = "JSON of image scan inference output."
+  }
+  primary_key {
+    columns = [column.scan_id, column.scan_inference_id]
+  }
+  foreign_key "scan_id" {
+    columns     = [column.scan_id]
+    ref_columns = [table.scan_v1.column.scan_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+}
+table "scan_v1" {
+  schema = schema.public
+  comment = "Stores the compressed images to be scanned and inferenced."
+  column "project_id" {
+    null = false
+    type = bigint
+  }
+  column "scan_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  column "image" {
```

add another field to store which image compression standard is used?
```suggestion
  column "image_content" {
```